Assets/MixedRealityToolkit/Utilities/Editor/DependencyWindow.cs

```diff
@@ -0,0 +1,339 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System;
+using System.Collections.Generic;
+using System.IO;
+using UnityEditor;
+using UnityEngine;
+
+using Object = UnityEngine.Object;
+
+namespace Microsoft.MixedReality.Toolkit.Utilities.Editor
+{
+    public class DependencyWindow : EditorWindow
+    {
+        private Object assetSelection = null;
+        private int maxDisplayDepth = 8;
+        private float assetGraphRefreshTime = 0.0f;
+        private Vector2 scrollPosition = Vector2.zero;
+
+        private class AssetGraphNode
+        {
+            public string guid = null;
+            public HashSet<AssetGraphNode> assetsThisDependsOn = new HashSet<AssetGraphNode>();
+            public HashSet<AssetGraphNode> assetsDependentOnThis = new HashSet<AssetGraphNode>();
+        }
+
+        private Dictionary<string, AssetGraphNode> dependencyGraph = new Dictionary<string, AssetGraphNode>();
+
+        private const string windowTitle = "Dependency Window";
+        private const string guidPrefix = "guid: ";
+        private const string nullGuid = "00000000000000000000000000000000";
+        private const int guidCharacterCount = 32;
+
+        private readonly string[] assetsWithDependencies =
+        {
+            ".unity",
+            ".prefab",
+            ".asset",
+            ".mat",
+            ".anim",
+            ".controller"
+        };
+
+        [MenuItem("Mixed Reality Toolkit/Utilities/Dependency Window", false, 3)]
+        private static void ShowWindow()
+        {
+            var window = GetWindow<DependencyWindow>();
+            window.titleContent = new GUIContent(windowTitle, EditorGUIUtility.IconContent("d_EditCollider").image);
+            window.minSize = new Vector2(520.0f, 380.0f);
+            window.RefreshAssetGraph();
+            window.Show();
+        }
+
+        private void OnGUI()
+        {
+            MixedRealityEditorUtility.RenderMixedRealityToolkitLogo();
```

Comment:
with PR #5093 , I am moving the RenderMixedRealityToolkit into MixedRealityInspectorUtility. We had duplicate files all doing editor utils functions

Reply:
Ah, thank you for the info. Updated!